src/__experimental__/components/select/select.component.js

```diff
@@ -51,6 +51,8 @@ class Select extends React.Component {
     ])
   }
 
+  static defaultProps = {}
```

Comment:
Possibly could do with a comment to indicate why empty `defaultProps` is included. (I think this was for Storybook?)